app/src/main/java/io/github/droidkaigi/confsched/viewmodel/ViewModel.java

```diff
@@ -0,0 +1,7 @@
+package io.github.droidkaigi.confsched.viewmodel;
+
+public interface ViewModel {
+
+    void destroy();
```

Comment:
ViewModel sometimes have to clean up some properties. For example rx.Subscription.
So I added `destroy` method to clean up.